src/main/java/com/openmrs/joiners/service/PersonService.java

```diff
@@ -20,4 +21,11 @@ public List<Person> getAllPersons(){
     public Person savePerson(Person person) {
         return personRepository.save(person);
     }
+
+    public Person updatePerson(Person updatedPerson){
```

Comment:
@Timug-123 this method does not reflect a proper Update operation, can you look into the Hibernate saveOrUpdate method or a related implementation as per the JpaRepository?